app/controllers/follows_controller.rb

```diff
@@ -0,0 +1,27 @@
+class FollowsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_question, only: [:create, :destroy]
+  before_action :set_follow, only: :destroy
+
+  respond_to :js
+
+  def create
+    respond_with @follow = @question.follows.create(user: current_user)
+    authorize! :create, @follow
```

Comment:
Do you understand that this authorization is useless? You've already created a follow. You should authorize before the action.

It this case it make sense to authorize question itself using some custom action, i.e. `authorize! :subscribe, @question`.